### PR TITLE
Bugfix/return object

### PR DESCRIPTION
**Pull request by thedevofthedevil666**
There were some errors when returning the results using INCHES convertion
This was solved by passing the array to a list

### CODE REVIEW

File: app/.~lock.new_jeans_dataset.csv# [hunk 1]
```diff
@@ -1 +1 @@
-,gabriel,devilpc,31.05.2022 14:38,file:///home/gabriel/.config/libreoffice/4;
```
Comment (kbetancourt3000) [inline app/.~lock.new_jeans_dataset.csv#:1]: add this file to .gitignore future pr

File: app/main.py [hunk 2]
```diff
@@ -43,35 +49,88 @@ def get_columns():
...
-            jeans = df['NewJeans'].to_numpy()
-            sizeFinal = jeans[index]
-            print(sizeFinal)
-            return sizeFinal
-        else:
-            raise HTTPException(status_code=400, detail="La talla no existe, favor de comunicarse con algún asesor")
-    else:
-        raise HTTPException(status_code=404, detail="No hay o existe una opcion por favor elija una para continuar")
+
+    match size.type:
```
Comment (kbetancourt3000) [inline app/main.py:53]: replace python version 3.9 instead 3.10, then replace match with other method

**Review by kbetancourt3000 — CHANGES_REQUESTED** [review]

**Review by kbetancourt3000 — CHANGES_REQUESTED** [review]
- be carefull in the .gitignore
- replace python version

File: app/.~lock.new_jeans_dataset.csv# (hunk 1 above)
Comment (thedevofthedevil666) [reply app/.~lock.new_jeans_dataset.csv#:1]: you mean dockerignore?

**Pull request merged**